### PR TITLE
Check for main component's existence before getting id

### PR DESCRIPTION
**Pull request by pjonori1**
Ran into a runtime error that this line fixed.

### CODE REVIEW

**Comment by pjonori1** [comment]
Hey @rlingineni and @protodave - long time, no chat! Just sent along my first PR!

File: src/index.ts [hunk 1]
```diff
@@ -352,7 +352,8 @@ export class FigmaCalculator extends FigmaDocumentParser {
     const isLibraryComponent = (instanceNode: any) => {
       // if it's a web file, then check the componentId else the mainComponent property to get the key
       const componentKey =
-        instanceNode.componentId || instanceNode.mainComponent.key;
+        instanceNode.componentId ||
+        (instanceNode.mainComponent ? instanceNode.mainComponent.key : 0);
```
Comment (protodave) [inline src/index.ts:356]: Hey, thanks for the PR! Can you check if this syntax would also address your runtime error?

```suggestion
        instanceNode.componentId || instanceNode.mainComponent?.key;
```

Comment (pjonori1) [reply src/index.ts:356]: That's much nicer. Testing now.

**Comment by jbenning-ic** [comment]
Hi @protodave, any ETA on when this can be merged/released? 😄

**Comment by pjonori1** [comment]
Dave, you’re the best. Thanks buddy!

**Pull request merged**